src/pages/account/story/StoryEdit.js

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from "react";
+import { Redirect, useParams } from "react-router-dom";
+import axios from "axios";
+
+import axiosApi from "../../../utils/AxiosApi";
+import FieldSet from "../../../components/Form/Fieldset";
+import Input from "../../../components/Form/Input";
+import TextArea from "../../../components/Form/TextArea";
+import Button from "../../../components/Button";
+import NavBar from "../../../components/NavBar";
+
+const StoryEdit = () => {
+  const [input, setInput] = useState({});
+  const [story, setStory] = useState("");
+  const [loading, setLoading] = useState(false);
+  const [redirect, setRedirect] = useState(false);
+  const [mediaPreview, setMediaPreview] = useState("");
+
+  const { storyId } = useParams();
+
+  useEffect(() => {
+    const fetchStoryData = async () => {
+      const result = await axiosApi.get(`/storyDetails/${storyId}`);
+      const singleStory = await result.data;
+      setStory(singleStory);
+    };
+    fetchStoryData();
+  }, [storyId]);
+
+  const handleImageUpload = async () => {
+    const data = new FormData();
+    data.append("file", input.media);
```

Comment:
Don't update the state like this.
If you need this url or something, save it to its own state

Reply:
It's not an URL. It's grabbing data I need from whatever you put in the input.media so it can be transformed to an URL via cloudinary